appinfo/routes.php

```diff
@@ -22,7 +22,7 @@
 return [
 	'routes' => [
 		['name' => 'site#showDefaultPage', 'url' => '/', 'verb' => 'GET'],
-		['name' => 'site#showPage', 'url' => '/{id}', 'verb' => 'GET'],
+		['name' => 'site#showPage', 'url' => '/{id}/{path}', 'verb' => 'GET', 'requirements' => ['path' => '.*']],
```

Comment:
Requirements don't need to be set

Reply:
Is this really equivalent? Not having `requirements`/ a regex at all is equivalent to matching anything?